### PR TITLE
Made more.lua command accept more keys

### PR DESCRIPTION
**Pull request by jamesb6626**
Added more keys for the command to recognise ("page down" goes to the next page in the same way as "space", "enter" and "down" print just the next line)
Moved the code to print lines from the file to a function to reduce the space used by repeating it for the new keys

### CODE REVIEW

**Comment by fnuecke** [comment]
Thanks, will test tomorrow.

File: src/main/resources/assets/opencomputers/loot/OpenOS/bin/more.lua [hunk 1]
```diff
@@ -5,6 +5,28 @@ local term = require("term")
...
+    end
+    local wrapped
+    wrapped, line = text.wrap(text.detab(line), w, w)
+    io.write(wrapped .. "\n")
+    i = i + 1
+  end
+  term.setCursor(1, h)
+  term.write(":")
+  term.setCursorBlink(true)
+  return 0
```
Comment (fnuecke) [inline src/main/resources/assets/opencomputers/loot/OpenOS/bin/more.lua]: One question right away, though, the return value is used in a boolean check on line 48.
Are you aware that `not not 0 == true` in Lua? I.e. that zero is _not_ false as it is in, say, C?

**Comment by jamesb6626** [comment]
Fixed this, thank you.
I didn't make a change to this that I made when testing it.
This should work now, I hope.

**Comment by fnuecke** [comment]
Thanks :-)

**Pull request merged**